p2p/peerManager/peerManager.go

```diff
@@ -447,7 +447,7 @@ func (pm *BasicPeerManager) reportPeerStats() {
 		q := query.Query{
 			Prefix: "/peers",
 		}
-		ticker := time.NewTicker(1 * time.Minute)
+		ticker := time.NewTicker(1 * time.Second)
```

Comment:
isnt this too much ?

Reply:
Yes I forgot to drop that commit for testing. Fixed now